src/routes/AddTrip/index.tsx

```diff
@@ -24,24 +51,29 @@ const AddTrip = () => {
             className="appearance-none block w-full bg-gray-200 text-gray-700 border rounded py-3 px-4 mb-3 leading-tight focus:outline-none focus:bg-white mb-4 text-center"
             type="text"
             placeholder="name your trip"
+            onChange={onChangeHandler}
+            name="title"
           />
         </label>
 
-        <label>
-          <input
-            className="appearance-none block w-full bg-gray-200 text-gray-700 border rounded py-3 px-4 mb-3 leading-tight focus:outline-none focus:bg-white mb-4 text-center"
-            type="text"
-            placeholder="trip date: yyyy-mm-dd"
-          />
-        </label>
-
-        <label>
-          <input
-            className="appearance-none block w-full bg-gray-200 text-gray-700 border rounded py-3 px-4 mb-3 leading-tight focus:outline-none focus:bg-white mb-4 text-center"
-            type="text"
-            placeholder="travel method"
-          />
-        </label>
+        <div className="inline-block relative w-full">
+          <select
+            className="block appearance-none w-full text-center pr-16 bg-gray-200 text-gray-700 border hover:border-gray-500 px-4 py-3  rounded  leading-tight focus:outline-none focus:shadow-outline"
+            onChange={onChangeHandler}
+            name="travel_method"
+          >
+            <option>Select your travel method</option>
+            <option>airplane</option>
+            <option>car</option>
+            <option>train</option>
+            <option>other</option>
```

Comment:
Let's make an array of methods and map it to these options, to shorten the overall file length